deployment/terraform/iam.tf

```diff
@@ -179,9 +179,9 @@ resource "aws_iam_role_policy_attachment" "spot_fleet_policy" {
 }
 
 #
-# Lambda resources
+# Lambda IAM resources
 #
-data "aws_iam_policy_document" "alert_batch_failures_assume_role" {
+data "aws_iam_policy_document" "lambda_assume_role" {
```

Comment:
Generalize the name of this policy document so that it can be used by the IAM role for the second Lambda function.